src/app/ui/button/button.component.ts

```diff
@@ -7,7 +7,6 @@ import { ButtonColor, ButtonSize } from './button.types';
   styleUrls: ['./button.component.scss'],
 })
 export class ButtonComponent {
-  @Input('text') value: string = 'text';
   @Input('size') size: ButtonSize = 'medium';
   @Input('color') color: ButtonColor = 'primary';
```

Comment:
Definitely not a fan of the Input aliases

Reply:
what's wrong tho?

Reply:
they are not needed in that case, there is nothing wrong tho, this is why i pressed the approve button